tests/unit/verticals/hmsl/test_hmsl_client.py

```diff
@@ -34,10 +36,18 @@ def hashes():
     return [hash_string(letter) for letter in "abcdefghijklmnopqrstuvwxyz"]
 
 
-@pytest.fixture
-def hmsl_client():
+def get_hmsl_client(
+    hmsl_command_path: str | None = None,
```

Comment:
type must be `Optional[str]` because of python < 3.10 :/

Reply:
Fixed in [d4056c5](https://github.com/GitGuardian/ggshield/pull/758/commits/d4056c53e8f277e55f329440531be6a4a676fe1d)

Reply:
There was also another case where I used `removeprefix` which was added in 3.9, I replaced it in [e3367d2](https://github.com/GitGuardian/ggshield/pull/758/commits/e3367d21f3a7ba172d964070719943e678185e1f)